src/eba_view.eliom

```diff
@@ -190,4 +190,191 @@ let password_form ?a ~service () =
          string_input ~input_type:`Submit ~value:"Send" ()
        ])
     ()
- }}
+
+let multiple_email_div_id = "multiple_email"
+}}
+{server{
+let rpc_emails_and_params_of_userid =
+  server_function Json.t<int64> Eba_user.emails_and_params_of_userid
+
+let rpc_add_email_to_user =
+  let add_email (user, email) =
+      Eba_user.add_email_to_user user email
+  in
+  server_function Json.t<int64 * string> add_email
+
+let rpc_update_users_primary_email =
+  server_function Json.t<string> Eba_user.update_users_primary_email
+
+let rpc_delete_email =
+  server_function Json.t<string> Eba_user.delete_email
+
+let rpc_send_mail_confirmation =
+  server_function Json.t<string> Eba_user.send_mail_confirmation
+}}
+
+{client{
+
+module ReactList = struct
+    let list t =
+      let open ReactiveData.RList in
+      make_from
+        (React.S.value t)
+        (React.E.map (fun e -> Set e) (React.S.changes t))
```

Comment:
This functionality is provided by `make_from_s`, and I am not sure it is a good idea to replicate stuff for compatibility with old releases.